recipes/fig2dev/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -x
+
+# Point to the right pkgconfig
+export PKG_CONFIG_LIBDIR="$PREFIX/lib/pkgconfig"
+# Add the build-dir's lib dir to the linker flags
+export LDFLAGS="-L$PREFIX/lib"
+# Add the build-dir's incude dir to the compiler flags
+export CFLAGS="-I$PREFIX/include"
```

Comment:
Please update this to be `export CFLAGS="${CFLAGS} -I$PREFIX/include"` to avoid overwriting what is set in the toolchain activate.sh